policy/runtime.go

```diff
@@ -242,3 +243,15 @@ func (r *PURuntime) SetServices(services []common.Service) {
 		r.options.Services = services
 	}
 }
+
+// PortMap returns the mapping from host port->container port
+func (r *PURuntime) PortMap() map[nat.Port][]string {
+	r.Lock()
+	defer r.Unlock()
+
+	if r.options != nil {
+		return r.options.PortMap
+	}
+
+	return nil
+}
```

Comment:
Please enhance the unit tests for the new methods.

Reply:
Done.